conjur/util/init_utils.py

```diff
@@ -39,3 +42,21 @@ def validate_init_action_ssl_verification_input(ca_path, is_self_signed, ssl_ver
         except Exception:
             # pylint: disable=raise-missing-from
             raise InvalidFilePermissionsException(f"No read access for: {ca_path}")
+
+
+def get_ssl_verification_meta_data_from_cli_params(ca_path, is_self_signed,
+                                                   ssl_verify) -> SslVerificationMetadata:
```

Comment:
If line separation is needed between params, do it between each param:

```suggestion
def get_ssl_verification_meta_data_from_cli_params(ca_path,
                                                  is_self_signed,
                                                   ssl_verify) -> SslVerificationMetadata:
```

Reply:
Done